mne/annotations.py

```diff
@@ -977,3 +1025,56 @@ def events_from_annotations(raw, event_id="auto",
                 (list(event_id_.keys()),))
 
     return events, event_id_
+
+
+@verbose
+def annotations_from_events(events, sfreq, event_desc=None, first_samp=0,
+                            orig_time=None, verbose=None):
+    """Convert an event array to an Annotations object.
+
+    Parameters
+    ----------
+    events : ndarray, shape (n_events, 3)
+        The events.
+    sfreq : float
+        Sampling frequency.
+    event_desc : dict | callable | None
```

Comment:
can you call it event_id ?

Reply:
2 reasons I didn't name it `event_id`:
- it's the opposite of what is used as `event_id` throughout MNE
- there is already a `_check_event_id` function in the file, which does the opposite of what I want to do in my case

But I'm happy to change if you think it's clearer

Reply:
good point. I had not realised it was a reversed event_id. Fine to keep it as is then